### DISCUSSION TITLE
Why the JRule class.

### DISCUSSION THREAD

**Discussion by kuimovvg** [seaside1/jrule#179] [General]
My question is, why do I need a Jrule class? Why is it not possible to mark classes with rules with an annotation, for example @jJrule, and take all classes from the jar file marked with an annotation? In many methods, the successor object from Jrule and the method of the same object are dragged. It turns out and confusion and you can pass an object one, and a method from another class. Also, in my opinion, this is an unnecessary binding. And methods from the Jrule class can be moved to the service class. 
I understand that deleting the Jrule class is a big refactoring.

**Comment by querdenker2k** [comment]
So the option would be to Inject a JRule instance via annotation to a rule class?
